standalone/src/models/store/store/store-draft/presets/empty.spec.ts

```diff
@@ -1,8 +1,39 @@
-import { TStoreDraft } from '../../types';
-import empty from './empty';
+import { TStoreDraftGraphql, TStoreDraftRest } from '../../types';
+import { emptyDraftRest, emptyDraftGraphql, empty } from './empty';
 
 it('should set all specified fields to undefined', () => {
-  const emptyStoreDraft = empty().build<TStoreDraft>();
+  const emptyStoreDraft = empty().build<TStoreDraftRest | TStoreDraftGraphql>();
+  expect(emptyStoreDraft).toMatchObject({
+    key: expect.any(String),
+    name: undefined,
+    languages: undefined,
+    countries: undefined,
+    distributionChannels: undefined,
+    supplyChannels: undefined,
+    productSelections: undefined,
+    custom: undefined,
+  });
+});
+
+it('should set all specified fields to undefined in graphql', () => {
+  const emptyStoreDraft =
+    emptyDraftGraphql().buildGraphql<TStoreDraftGraphql>();
```

Comment:
No need for the type as we are already building from the graphql version of the mode.
```suggestion
  const emptyStoreDraft = emptyDraftGraphql().build();
```